vars/provisionNodes.groovy

```diff
@@ -113,14 +119,24 @@ def call(Map config = [:]) {
                               chmod 600 /localhome/jenkins/.ssh/{authorized_keys,id_rsa*}
                               chown -R jenkins.jenkins /localhome/jenkins/.ssh
                               echo \\"jenkins ALL=(ALL) NOPASSWD: ALL\\" > /etc/sudoers.d/jenkins
+                              yum -y install yum-utils
+                              pkgs=\\"${inst_rpms}\\"
```

Comment:
Why use a local variable here, rather than just using inst_rpms on L139?

Reply:
`$pkgs` is used on more than just L139. Yes, I could interpolate `inst_rpms` through all the places where `$pkgs` is used but it seems cleaner (and easier to lift this script to somewhere else if needed) to have a single interface point between the script and it's caller.

Reply:
Ah, I only saw the one instance.